pypesto/objective/__init__.py

```diff
@@ -10,6 +10,7 @@
 from .priors import NegLogPriors, NegLogParameterPriors
 from .util import res_to_chi2, sres_to_schi2
 from .base import ObjectiveBase
+from .aesara import AesaraObjective
```

Comment:
I think this is not desirable, as it requires an installation of aesara

Reply:
ok, should i then remove it as well from read/write? could also do a `try...except ImportError`.